tools/install.go

```diff
@@ -84,7 +86,7 @@ func EnsureKind() {
 
 // Install kind at the specified version
 func EnsureKindAt(version string) {
-	if ok, _ := pkg.IsCommandAvailable("kind", ""); ok {
+	if ok, _ := pkg.IsCommandAvailable(filepath.Join(gopath.GetGopathBin(), "kind"), ""); ok {
```

Comment:
This should work as it was originally. The magex.pkg package will check if kind is available on PATH first, verify a version match if specified (in this case it's not), and then use go install to install it into your GOPATH/bin directory which should be on the PATH of a go development environment.

What did you run into that prompted the change?

Reply:
I had a older version of kind install in /usr/local/bin.  The magefile was installing another version in $HOME/go/bin.  PATH was set to look in /usr/local/bin before $HOME/go/bin

The test was failing for me.  I can remove this and keep this focused on the XBuildAll change.  Maybe just manage the PATH in the failing test

Reply:
Thanks for the extra detail! I was able to reproduce after I changed my PATH to have GOPATH/bin after /usr/local/bin. I always prepend to path so that I can override system installations and hadn't considered what the code would do when configured that other way.

I just added functionality to install tools into a local bin to the magex library. So I'll take care of this in a separate PR this week. Can you remove this check and we'll track this in #8?

Reply:
I just added a minor change to ensure the current test had the right path expectations.  We're looking at more helper functions to manage explicit project deps relative to the project.  You never know what you'll get from one user to the next.